omim2obo/main.py

```diff
@@ -202,11 +202,19 @@ def omim2obo(use_cache: bool = False):
         else:
             graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(label))))
 
+        # Reify on abbreviations. See: https://github.com/monarch-initiative/omim/issues/2
         exact_labels = [s.strip() for s in label.split(';')]
-        # the last string is an abbreviation. Add OWL reification. See issue #2
         if len(exact_labels) > 1:
             abbr = exact_labels.pop()
             graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbr)))
+            # Acronym: uppercase, no numbers, no whitespace, <10 chars
+            if abbr.isupper() and not any(char.isspace() for char in abbr) and len(abbr) < 10:
```

Comment:
This is not correct. The mimTitles.txt file indicates which values OMIM considers as symbols and it is these values that OMIM designates as symbols that will be tagged as abbreviations in the resulting omim.owl file.

The headers are:
`Prefix	MIM Number	Preferred Title; symbol	Alternative Title(s); symbol(s)	Included Title(s); symbols`

Reply:
Yes, that is basically correct. However we have this bug:
- https://github.com/monarch-initiative/omim/issues/126

In `mimTitles.txt`, all of the titles and synonyms are uppercase.
But our pipeline has various rules by which it adjusts the capitalization. I believe that there are bugs. So if there are indeed bugs as a temporary failsafe, I think we probably want to (a) not add the lowercase ones, fix the bug, and then add them, rather than (b) add them now as lowercase, but then they get sync'ed into Mondo and we have to fix them there.

If you do want to go with (b), though, let me know and I'll remove this line now.

If we go with (a), then this line stays here until I address #126.

Reply:
In `mimTitles.txt` the "symbol" value from the OMIM column "Preferred Title; symbol" should be parsed out from that column and the "symbol" value should not have any changes in it's capitalization as it's found in the file.

Reply:
Sounds good. I forgot (c), which we'll go with: Fix #126 first.

Reply:
We discussed today, and #126 addresses. #126 removes this one line as well. It is not needed.